keycloak-admin.gemspec

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.3'
 
+  spec.add_dependency "activesupport"
```

Comment:
This library is intended to not depend on such libraries.
This is not supposed to always be used in Rails project, for example, but also simple ruby scripts.